text/0026-semantic-versioning.md

```diff
@@ -0,0 +1,109 @@
+# Semantic Versioning of Buildpacks and Builders
+
+## Summary
+
+All Paketo buildpack releases use the semver syntax for versioning, but does not necessarily follow semver semantics.
+
+The [CNB Buildpacks spec](https://github.com/buildpacks/spec/blob/buildpack/v0.6/buildpack.md) says:
+
+> The buildpack version:
+>
+> - MUST be in the form <X>.<Y>.<Z> where X, Y, and Z are non-negative integers and must not contain leading zeros.
+>    - Each element MUST increase numerically.
+>    - Buildpack authors will define what changes will increment X, Y, and Z.
+
+
+It does not define what changes will increment the Major (`X`), Minor (`Y`) and Patch (`Z`) versions
+of buildpack releases.
+
+This RFC attempts to set guidelines and good practices for Paketo Buildpack authors/maintainers to follow
+when a buildpack release has to be created.
+
+## Motivation
+
+As a Buildpack/Builder maintainer,<br/>
+I need a release versioning guideline,
+so I can make my release versions consistent.
+
+As a Buildpack/Builder user,<br/>
+I want to know how versioning is done,
+so I can use the appropriate version and
+decide when to upgrade.
```

Comment:
Not 100% sure it matters, cause I see plenty of reason to do this, but I would say another motivation is that "As a Buildpack/Builder user, I want versioning to be consistent across all Paketo buildpacks so I don't have to understand multiple policies."

Reply:
Done